src/urllib3/connectionpool.py

```diff
@@ -428,9 +432,9 @@ def _make_request(
         # urllib3.request. It also calls makefile (recv) on the socket.
         try:
             if chunked:
-                conn.request_chunked(method, url, **httplib_request_kw)
+                conn.request_chunked(method, url, body, headers)
```

Comment:
Let's use keyword arguments for `body` and `headers`

Reply:
done